src/main/webapp/resources/js/pages/analysis/components/AnalysisBioHansel.jsx

```diff
@@ -67,57 +57,88 @@ export default function AnalysisBioHansel() {
-                <Menu.Item key="file_preview">
-                  <Link to={`${DEFAULT_URL}/${BIOHANSEL.FILE_PREVIEW}`}>
-                    {i18n("AnalysisOutputs.outputFilePreview")}
-                  </Link>
-                </Menu.Item>
-              </Menu>
-            );
-          }}
-        </Location>
-      </Sider>
+  const biohanselResults = [
+    {
+      title: i18n("AnalysisBioHansel.sampleName"),
+      desc: bioHanselResults ? bioHanselResults.sample : ""
+    },
+    {
+      title: `${i18n("AnalysisBioHansel.schemeName")} (v${i18n(
+        "AnalysisBioHansel.schemeVersion"
+      )})`,
+      desc: bioHanselResults
+        ? `${bioHanselResults.scheme} (${bioHanselResults.scheme_version})`
+        : ""
+    },
+    {
+      title: i18n("AnalysisBioHansel.subtype"),
+      desc: bioHanselResults ? bioHanselResults.subtype : ""
+    },
+    {
+      title: i18n("AnalysisBioHansel.averageTileFrequency"),
+      desc:
+        bioHanselResults && bioHanselResults.avg_tile_coverage
+          ? bioHanselResults.avg_tile_coverage.toString()
+          : ""
+    },
+    {
+      title: i18n("AnalysisBioHansel.qualityControlStatus"),
+      desc: bioHanselResults ? (
+        bioHanselResults.qc_status === "PASS" ? (
+          <Success
+            message={bioHanselResults.qc_status}
+          />
+        ) : bioHanselResults.qc_status === "FAIL" ? (
+          <Error
+            message={formatQcMessage(
+              bioHanselResults.qc_status,
+              bioHanselResults.qc_message
+            )}
+          />
+        ) : (
+          <Warning
+            message={formatQcMessage(
+              bioHanselResults.qc_status,
+              bioHanselResults.qc_message
+            )}
+          />
+        )
+      ) : (
+        ""
+      )
+    }
+  ];
 
-      <Layout style={{ paddingLeft: SPACE_MD, backgroundColor: grey1 }}>
-        <Content>
-          {typeof bioHanselResults === "undefined" ? (
-            <WarningAlert
-              message={i18n("AnalysisBioHansel.resultsUnavailable")}
-            />
-          ) : bioHanselResults !== null ? (
-            <Suspense fallback={<ContentLoading />}>
-              <Router>
-                <BioHanselInfo
-                  bioHanselResults={bioHanselResults}
-                  path={`${DEFAULT_URL}/${BIOHANSEL.INFO}`}
-                  default
-                />
-                <OutputFilePreview
-                  path={`${DEFAULT_URL}/${BIOHANSEL.FILE_PREVIEW}`}
-                />
-              </Router>
-            </Suspense>
-          ) : (
-            <ContentLoading />
-          )}
-        </Content>
-      </Layout>
+  /*
+   * Formats the QC status and QC messages to display
+   * in a list.
+   */
+  function formatQcMessage(qc_status, qc_message) {
+    let msgs = [];
+    if (qc_message) {
+      msgs = qc_message.trim().split("|");
+    }
+
+    return (
+      <>
+        <span>{qc_status}</span>
+        <br />
```

Comment:
You needed the `br` element because you used a `span` (inline) element.  If you used a `div` (block) element the break would already be there.